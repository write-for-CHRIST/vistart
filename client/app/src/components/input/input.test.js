import 'react-native'
import React from 'react'
import Input from './'

import renderer from 'react-test-renderer'

test('renders without crashing', () => {
  const tree = renderer.create(<Input placeholder="Hello World" />).toJSON()
  expect(tree).toMatchSnapshot()
})
