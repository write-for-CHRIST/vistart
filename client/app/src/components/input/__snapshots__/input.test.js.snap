// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<View
  style={
    Array [
      Object {
        "borderRadius": 10,
        "width": "100%",
      },
      undefined,
    ]
  }
>
  <TextInput
    allowFontScaling={true}
    placeholder="Hello World"
  />
</View>
`;
